sdk/resourcemanager/azure-resourcemanager-containerinstance/src/main/java/com/azure/resourcemanager/containerinstance/models/ContainerGroup.java

```diff
@@ -932,28 +932,64 @@ WithContainerInstanceAttach<ParentT> withReadOnlyVolumeMountSetting(
             interface WithLivenessProbe<ParentT> {
                 /**
                  * Specifies the container's liveness probe to execute a given command at a given interval.
+                 * <p>If the probe command execution fails, the container will restart.</p>
+                 * <p>Only 1 liveness probe can be set for a given container.</p>
                  *
                  * @param command the command for the probe to execute
                  * @param probePeriodSeconds the interval at which the command executes
                  * @return the next stage of the definition
+                 * @see <a href="https://learn.microsoft.com/en-us/azure/container-instances/container-instances-liveness-probe#liveness-command">liveness command</a>
+                 * @see <a href="https://learn.microsoft.com/en-us/azure/container-instances/container-instances-liveness-probe#liveness-probes-and-restart-policies">liveness probes and restart policies</a>
                  */
-                WithContainerInstanceAttach<ParentT> withLivenessProbeExec(List<String> command, int probePeriodSeconds);
+                WithContainerInstanceAttach<ParentT> withLivenessProbeExecutionCommand(List<String> command, int probePeriodSeconds);
+
+                /**
+                 * Specifies the container's liveness probe to execute a given command at a given interval.
+                 * <p>After the probe command execution failure reaches the given threshold, the container will restart.</p>
+                 * <p>Only 1 liveness probe can be set for a given container.</p>
+                 *
+                 * @param command the command for the probe to execute
+                 * @param probePeriodSeconds the interval at which the command executes
+                 * @param failureThreshold the failure threshold before the container restarts
```

Comment:
at least make clear in doc, what threshold? seconds? count?

Reply:
Makes sense. I think it's the consecutive failures in a row, paring with Kubernetes's threshold. I'll double-check. Thanks!

Reply:
Added verfication for failureThreshold. It is indeed consecutive failures in a row.